scripts/generate_json_docs.py

```diff
@@ -295,11 +296,14 @@ def build_type(type_id, title, contents):
 
 
 def clean_source_path(source):
```

Comment:
The goal is to get the GitHub url for that file. However the doc site
prepends the repo URL. So you need just /gcloud/../../...py

On Wednesday, August 3, 2016, Danny Hermes <notifications@github.com> wrote:

> In scripts/generate_json_docs.py
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/2052#discussion_r73431159>
> :
>
> > @@ -295,11 +296,14 @@ def build_type(type_id, title, contents):
> >
> >
> >  def clean_source_path(source):
>
> What is the point of this function? Please provide a docstring and then we
> can discuss the implementation. AFAICT the goal is to determine the true
> path to a module and there are better ways to do this that don't involve
> the sys.path guesswork.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/2052/files/10191021d01b96ab7d97045aaab0f7326eae5a13#r73431159>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALvyH2QSFmRiDkDGZZgpcOZDgxQCg9yks5qcRHtgaJpZM4JcF9G>
> .
>


-- 
Tom Schultz